build/CMakeLists.txt

```diff
@@ -116,11 +116,7 @@ ExternalProject_Add(protobuf
 
 # Location where protobuf-config.cmake will be installed varies by
 # platform
-if (WIN32)
-  set(_FINDPACKAGE_PROTOBUF_CONFIG_DIR "${CMAKE_CURRENT_BINARY_DIR}/protobuf/cmake")
-else()
-  set(_FINDPACKAGE_PROTOBUF_CONFIG_DIR "${CMAKE_CURRENT_BINARY_DIR}/protobuf/lib/cmake/protobuf")
-endif()
+set(_FINDPACKAGE_PROTOBUF_CONFIG_DIR "${CMAKE_CURRENT_BINARY_DIR}/protobuf/${CMAKE_INSTALL_LIBDIR}/cmake/protobuf")
```

Comment:
This is almost the same change made in the PR #742, which was reverted as part of PR #763, possibly by rebasing.